packages/module-manager/src/types/OctoModule.ts

```diff
@@ -0,0 +1,11 @@
+import type { ReactElement } from 'react';
+
+import type { PostMode } from './PostModes';
+import type { Service } from './Service';
+
+export interface OctoModule {
```

Comment:
A gente deixou de usar I e T pra pra interface e modulo por qual motivação?

Reply:
Esses tipos são utilizados por outros desenvolvedores fora do Octopost para desenvolver módulos, e o padrão de usar sufixos ou prefixos não é muito bem visto.

O próprio time do Typescript não curte muito:
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines

https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript

https://twitter.com/housecor/status/1653006517212151808

https://ts.dev/style/#identifiers

https://passionfordev.com/you-dont-need-to-prefix-interfaces-in-typescript-with-i/

https://til.hashrocket.com/posts/cfnlyzxcrc-do-not-prefix-typescript-interface-names

https://twitter.com/bterlson/status/1081959796398469120